Gems/Multiplayer/Code/Source/NetworkEntity/NetworkSpawnableLibrary.h

```diff
@@ -23,12 +23,15 @@ namespace Multiplayer
         NetworkSpawnableLibrary();
         ~NetworkSpawnableLibrary();
 
-        /// INetworkSpawnableLibrary overrides.
+        //! INetworkSpawnableLibrary overrides.
+        //! @{
+        // Iterates over all assets (on-disk and in-memory) and stores any spawnables that are "network.spawnables"
+        // This allows us to look up network spawnable assets by name or id for later use
```

Comment:
nit: Avoid personal pronouns in comments. Not clear to external readers who 'us' is